http://bugs.gentoo.org/202511

2007-12-24  Mike Frysinger  <vapier@gentoo.org>

	* elf/Makefile ($(objpfx)ld.so): Use POSIX (EF)? in awk.

--- libc/elf/Makefile
+++ libc/elf/Makefile
@@ -308,7 +308,7 @@
 		  -Wl,-soname=$(rtld-installed-name) -T $@.lds
 	rm -f $@.lds
 	readelf -s $@ \
-	  | awk '($$7 ~ /^UND(|EF)$$/ && $$1 != "0:" && $$4 != "REGISTER") { print; p=1 } END { exit p != 0 }'
+	  | awk '($$7 ~ /^UND(EF)?$$/ && $$1 != "0:" && $$4 != "REGISTER") { print; p=1 } END { exit p != 0 }'
 
 # interp.c exists just to get this string into the libraries.
 CFLAGS-interp.c = -D'RUNTIME_LINKER="$(slibdir)/$(rtld-installed-name)"' \
